doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+(first-steps)=
+# First steps with LXD
+
+This tutorial guides you through the first steps with LXD.
+It covers installing and initializing LXD, creating and configuring some instances, interacting with the instances, and creating snapshots.
+
+After going through these steps, you will have a general idea of how to use LXD, and you're ready to explore more advanced use cases!
+
+## Install and initialize LXD
+
+The easiest way to install LXD is to install the snap package.
+If you prefer a different installation method, or use a Linux distribution that is not supported by the snap package, see {ref}`installing`.
+
+1. Install `snapd`:
+
+   1. Run `snap version` to find out if snap is installed on your system:
+
+      ```{terminal}
+      :input: snap version
+
+      snap    2.59.4
+      snapd   2.59.4
+      series  16
+      ubuntu  22.04
+      kernel  5.15.0-73-generic
+      ```
+
+      If you see a table of version numbers, snap is installed and you can continue with the next step of installing LXD.
+
+   1. If the command returns an error, run the following commands to install the latest version of ``snapd``:
```

Comment:
Since `apt` is Debian-specific, I'd probably add: "... on a Debian/Ubuntu machine:" Or maybe even point the reader to https://snapcraft.io/docs/installing-snapd which has the advantage of covering a lot of distros... and be maintained by other folks ;)